spec/src/main/asciidoc/circuitbreaker.asciidoc

```diff
@@ -38,7 +38,37 @@ succeed before the circuit can be closed. After the specified number of successf
 before the successThreshold is reached the circuit will transition to open. 
 
 Note that circuit state transitions will reset the Circuit Breaker's records. For example, when the circuit transitions to closed a new
-rolling failure window is created with the configured requestVolumeThreshold and failureRatio.
+rolling failure window is created with the configured requestVolumeThreshold and failureRatio. The circuit state will only be assessed when the rolling window reaches the `requestVolumeThreshold`. 
+ 
+[source, java]
+----
+@CircuitBreaker(successThreshold = 10, requestVolumeThreshold = 4, failureRatio=0.5, delay = 1000)
+public Connection serviceA() {
+   Connection conn = null;
+   counterForInvokingServiceA++;
+   conn = connectionService();
+   return conn;
+}
+----
+Below are a couple scenarios to demonstrate when the circuit opens for the above code snippet.
+* Scenario 1
+     Request 1 - success
+     Request 2 - failure
+     Request 3 - failure
+     Request 4 - success
+     Request 5 - CircuitOpenException
```

Comment:
This section does not render correctly in the spec pdf

Reply:
It seems my asciidoc preview rendered ok but not the generated one. fixed.